lib/git-fastclone.rb

```diff
@@ -355,6 +355,7 @@ def with_git_mirror(url)
           error_strings = [
             'missing blob object',
             'remote did not send all necessary objects',
+            'packed object',
```

Comment:
not _super_ happy on this message matching _but_ all of them start with `fatal:` so I am personally OK with that.

Reply:
What about `/packed object [a-z0-9]+ \(stored in .*?\) is corrupt/`?

Reply:
test did not pass updating a bit to make it pass.

Reply:
updated.